node/builder.go

```diff
@@ -514,8 +516,12 @@ func ConfigBoost(cfg *config.Boost) Option {
 		})),
 
 		// Lotus Markets
-		Override(new(lotus_dtypes.StagingBlockstore), lotus_modules.StagingBlockstore),
-		Override(new(lotus_dtypes.StagingGraphsync), lotus_modules.StagingGraphsync(cfg.LotusDealmaking.SimultaneousTransfersForStorage, cfg.LotusDealmaking.SimultaneousTransfersForStoragePerClient, cfg.LotusDealmaking.SimultaneousTransfersForRetrieval)),
+		Override(new(lotus_dtypes.StagingBlockstore), From(new(*piecedirectory.PieceDirectory))),
```

Comment:
I think we can remove this line as IIUC `lotus_dtypes.StagingBlockstore` is no longer being used anywhere